src/css/main.css

```diff
@@ -1 +1,119 @@
-/* General rules */
+*{
+    margin: 0;
+
+}
+header{
+    text-align: center;
+    height: 96vh;
+    border: 1px solid transparent;
+    background: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg) 50% 100% no-repeat;
+}
```

Comment:
Let's add this background property
```css
    background-size: cover;
```
This will allow the background image to cover all white space by the sides of the element when zooming in or out.